EnglishFilter/sections/antiadblock.txt

```diff
@@ -386,6 +386,8 @@ hclips.com#@#.adholder2
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/51775
 @@||100count.net/*/linko/advertisement.js
+! https://github.com/AdguardTeam/AdguardFilters/issues/51794
+1shortlink.com#%#//scriptlet("set-constant", "showAlertRequireDisableAdblocker", "noopFunc")
```

Comment:
You can do it without using a scriptlet because `$empty` modifier works fine with `adsbygoogle.js`.
But it doesn't work for content blockers and `Edge`, so use hints.